lib/ParamedicConfig.js

```diff
@@ -104,4 +117,44 @@ ParamedicConfig.prototype.getTimeout = function() {
     return DEFAULT_TIMEOUT;
 };
 
+ParamedicConfig.prototype.getLogPath = function() {
+    return this._config.logPath;
+};
+
+ParamedicConfig.prototype.setLogPath = function(logPath) {
+    this._config.logPath = logPath;
+};
+
+ParamedicConfig.prototype.getLogMins = function() {
+    return this._config.logMins;
+};
+
+ParamedicConfig.prototype.setLogMins = function(logMins) {
+    this._config.logMins = logMins;
+};
+
+ParamedicConfig.prototype.setAppName = function(appName) {
+    this._config.appName = appName;
```

Comment:
Either remove this config parameter & use a default value that you can assume to be true 
or create the app with the name specified here.